news_hash/NER.py

```
@@ -0,0 +1,49 @@
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+from collections import defaultdict
+
+key = "921242739dac433db87fc7edd6b719ca"
+endpoint = "https://nzx.cognitiveservices.azure.com/"
+
+def authenticate_client():
+    ta_credential = AzureKeyCredential(key)
+    text_analytics_client = TextAnalyticsClient(
+            endpoint=endpoint,
+            credential=ta_credential)
+    return text_analytics_client
+
+client = authenticate_client()
+text = "Cops released new footage of the man wanted for attacking a woman inside a Midtown subway station this week. The 23-year-old victim was standing on the northbound D platform at the 34th Street-Herald Square station around 4 a.m. Monday when a man punched her and put her in a headlock — throwing her to the floor, according to police. The New York City Police Department is seeking the public’s assistance identifying the following individual. Footage released by police shows the man hopping the turnstile at the Herald Square stop in Midtown on November 22, 2021. A woman was punched in the face, put in a headlock and robbed of her pocketbook on the northbound D platform at 34th Street/Herald Square on November 22, 2021. Surveillance footage shows the suspect wearing a dark-colored sweatshirt and sweatpants. While the victim was down, the mugger allegedly touched the woman’s private parts and took off with her purse, cops said. Police are still looking for the suspect, who is shown on surveillance footage released Monday night wearing a dark sweatshirt and sweatpants. The video shows him jumping the turnstile and walking through a desolate mezzanine area. The victim was taken to Lenox Hill Hospital in stable condition."
+
+
+def entity_recognition(text):
+    try:
+        documents = [text]
```

validate param before reference, otherwise the exception would potentially swallow the real issue.